onsp_co2_employee_commuting/models/res_company.py

```diff
@@ -0,0 +1,12 @@
+
+
+from odoo import models, fields
+
+class ResCompany(models.Model):
+    _name = "res.company"
+    _inherit = ["res.company", "carbon.mixin"]
```

Comment:
As `carbon.mixin` is inherited by `res.company` in the main module, you don't need to write it here also

```suggestion
    _inherit = "res.company"
```